try:
    print 5/0
# catch any exception
except Exception as e:
    print "Exception:", e.message